.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please include a summary of the changes made.
+
+### Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes a bug)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] feature Improvement (changes in an existing feature that improves it)
+- [ ] Breaking change (big change to existing code like re-design)
+- [ ] Clean (changes only to names/format/order and stuff like it)
+
+## Screenshots
+
+Please provide videos/photos if needed.
+
+## Checklist
+
+- [ ] I have performed a self-review of my code
```

Comment:
maybe add a format/spotless something idk

Reply:
There’s a check for that, no need